src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/PersonopplysningerService.kt

```diff
@@ -70,8 +70,8 @@ class PersonopplysningerService(
     }
 
     @Cacheable("personopplysninger", cacheManager = "shortCache")
-    fun hentPersonopplysninger(personIdent: String): PersonopplysningerDto {
-        val grunnlagsdata = grunnlagsdataService.hentFraRegisterForPersonOgAndreForeldre(personIdent, emptyList())
+    fun hentPersonopplysningerUtenVedtakshistorikk(personIdent: String): PersonopplysningerDto {
```

Comment:
Hvorfor endrer denne navn? Er det andre funksjoner som henter personopplysninger med vedtakshistorikk?

Kan virke smud nå fordi vi er vandt til at det kommer med vedtakshistorikk, men etterhvert har ingen et forhold til at det var med vedtakshistorikk. Da ville jeg tenkt om jeg var "ny i koden" at det finnes en som er _med_. Så om man beholder selve navnet så endrer man heller hva vi anser som personopplysninger og fra nå definerer at dette ikke inkluderer vedtakshistorikk.

Se bort fra denne om det er meningen at det finnes en _med_ vedtakshistorikk for da støtter jeg navnet

Reply:
Oi, denne skal være uten, og den over skal jo være MED 😂 Litt overivrig renaming ser jeg

Reply:
Den endrer navn fordi den har hentet med vedtakshistorikk, som de andre, men nå henter den uten vedtakshistorikken.